### DISCUSSION TITLE
worse temps

### DISCUSSION THREAD

**Discussion by ElfyDelphi** [Rem0o/FanControl.Releases#1118] [General]
I'm no expert with this stuff, just want to say that first. I've tried out this software a bit here and there over the past month and I've never been able to get any settings I'm happy with. I have an H7 Flow case with 3 140mm fans in the front, one in back for exhaust. 3080 Ti FTW3 Ultra w its fans being controlled via Precision, just had it on the "stealth" curve preset. The case fans were being controlled via bios.

I just loaded up a certain game that seems to run pretty hot. With the configuration listed above, it was maxing out at 78C, yet low noise still. This prompted me to wanna tinker with FanControl again. I pair the case fans curve to the GPU temp and even with case + gpu fans set more aggressively, the game went over 82C, even though the fans are spinning faster, and much louder than my bios/precision configuration. Stumped as to how this is possible. I'd think that even simply pairing the case fans to the GPU instead of them being locked to the CPU via bios would help a lot, but it hasn't.

**Comment by Rem0o** [comment]
Balancing your fan speeds in your case can be tricky. More speed won't necessarly give you lower temps.  There can be "hot pockets" of air being trapped and not exhaust properly.

I've found that exhaust in most case is more critical than intake. 3 to 1 never worked for me in my experience.

In any case ( no pun intended ), you have to keep experimenting and find the sweet spot.
